data/Rawhide/base-runtime/common-hints.txt

```diff
@@ -1,41 +1,24 @@
+clamav-data-empty
 coreutils
 cronie-noanacron
 curl-minimal
 environment-modules
-fedora-logos-httpd
 fedora-release
 glibc-minimal-langpack
-gnuplot
-infinipath-psm
 java-1.8.0-openjdk-javadoc
 kernel-core
 kernel-devel
 libcrypt
 libcurl-minimal
-libudisks2
-libudisks2-devel
 libverto-libev
-maven-resolver-api
-maven-resolver-connector-basic
-maven-resolver-impl
-maven-resolver-spi
-maven-resolver-transport-file
-maven-resolver-transport-http
-maven-resolver-transport-wagon
-maven-resolver-util
-pkgconf-pkg-config
-libdnf
```

Comment:
chosen by solver by default, but fix is in koji, so at some point it will disappear from alternatives